packages/compiler/src/compiler/chain.test.ts

```diff
@@ -101,7 +101,12 @@ describe('chain', async () => {
 
     const systemMessage = conversation.messages[0]!
     expect(systemMessage.role).toBe('system')
-    expect(systemMessage.content).toBe('System message')
+    expect(systemMessage.content).toEqual([
+      {
+        type: 'text',
+        text: 'System message',
+      },
```

Comment:
This is critical. Before we discarded system messages because from compiler we send only the first `content` item in a message as a string. Now we send an array of `content` items like in this text.